kolibri/plugins/facility/assets/src/views/users/common/UsersTable.vue

```diff
@@ -158,10 +158,9 @@
       @close="closeModal"
     />
 
-    <DeleteUserModal
+    <MoveToTrashModal
       v-if="modalShown === Modals.DELETE_USER"
-      :id="userToChange.id"
-      :username="userToChange.username"
+      :selectedUsers="new Set([userToChange.id])"
       @close="closeModal"
       @change="$emit('change')"
```

Comment:
Not sure how we can best handle this, but here we may need to emit a config object to set the `resetSelection` option to true, so the selection is reset after a delete [here](https://github.com/lianaharris360/kolibri/blob/f8828bc25d3debc1f294d2861a6212bf660dea39/kolibri/plugins/facility/assets/src/views/users/NewUsersPage.vue#L171), so in case the removed user was selected, we reset the selection, and avoid having operations with the removed users:

https://github.com/user-attachments/assets/2138b264-c389-42cb-a7a6-faef1af4fe43